ui/src/routes/_layout/runs/index.tsx

```diff
@@ -0,0 +1,340 @@
+import { FilterMultiSelect } from '@/components/data-table/filter-multi-select';
+import { LoadingIndicator } from '@/components/loading-indicator';
+import { OrtRunJobStatus } from '@/components/ort-run-job-status';
+import { RunDuration } from '@/components/run-duration';
+import { ToastError } from '@/components/toast-error';
+import { Badge } from '@/components/ui/badge';
+import { Button } from '@/components/ui/button';
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from '@/components/ui/card';
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipTrigger,
+} from '@/components/ui/tooltip';
+import { config } from '@/config';
+import { getStatusBackgroundColor } from '@/helpers/get-status-class';
+import { toast } from '@/lib/toast';
+import { ortRunStatus, paginationSchema, statusSchema } from '@/schemas';
+
+const defaultPageSize = 10;
+const pollInterval = config.pollInterval;
+
+const columns: ColumnDef<OrtRunSummary>[] = [
+  {
+    accessorKey: 'repoUrl',
+    header: 'Repository',
+    cell: function CellComponent({ row }) {
+      const { data: repo } = useRepositoriesServiceGetRepositoryById({
+        repositoryId: row.original.repositoryId,
+      });
+
+      const { data: org } = useOrganizationsServiceGetOrganizationById({
+        organizationId: row.original.organizationId,
+      });
+
+      const { data: prod } = useProductsServiceGetProductById({
+        productId: row.original.productId,
+      });
+
+      return (
+        <div>
+          <Link
+            className='block font-semibold text-blue-400 hover:underline'
+            to={
+              '/organizations/$orgId/products/$productId/repositories/$repoId'
+            }
+            params={{
+              orgId: row.original.organizationId.toString(),
+              productId: row.original.productId.toString(),
+              repoId: row.original.repositoryId.toString(),
+            }}
+          >
+            {repo?.url}
+          </Link>
+          <div className='text-xs italic text-slate-500'>
+            in{' '}
+            <Link
+              className='hover:underline'
+              to={'/organizations/$orgId'}
+              params={{ orgId: row.original.organizationId.toString() }}
+            >
+              {org?.name}
+            </Link>
+            /
+            <Link
+              className='hover:underline'
+              to={'/organizations/$orgId/products/$productId'}
+              params={{
+                orgId: row.original.organizationId.toString(),
+                productId: row.original.productId.toString(),
+              }}
+            >
+              {prod?.name}
+            </Link>
+          </div>
+        </div>
+      );
+    },
+  },
+  {
+    accessorKey: 'createdAt',
+    header: 'Created At',
+    cell: ({ row }) => (
+      <div>
+        {new Date(row.original.createdAt).toLocaleString(navigator.language)}
+      </div>
+    ),
+    size: 100,
```

Comment:
The default min-width for a column is 150 px so it lowers that, so that some of the columns are a bit narrower and the columns fit on the current width that we have on the layout. I still changed it for a couple of the columns cause I noticed the table was actually still overflowing even though it looked ok.